app/test-utils/testFunctions/questionTests/errorTests.js

```diff
@@ -129,28 +130,39 @@ const mandatoryErrorTest = ({
   }
 };
 
-export const runErrorTests = ({
+export const runErrorTests = async ({
   pageSetup,
   sectionManifest,
   questionId,
   sectionId,
   questionData,
   dashboardId,
 }) => {
-  Object.keys(questionData.errorResponse).forEach((errorType) => {
+  const errorPostBody = Object.keys(sectionManifest.questions).reduce((acc, question) => {
+    const questionType = sectionManifest.questions[question].type;
+    if (questionType === 'radiobutton-options') acc[question] = null;
+    else if (questionType === 'checkbox-options') acc[question] = [];
+    else if (questionType === 'bulletpoint-list') acc[question] = new Array(sectionManifest.questions[question].maxItems).fill('');
+    else acc[question] = '';
+    return acc;
+  }, {});
```

Comment:
why can't we just pass in from the ui.test the errorPostBody we want to return?

Reply:
I did start doing this but it is repetitive and predictable so why not automate it??